core/unit_test/TestResize.hpp

```diff
@@ -23,6 +23,9 @@ namespace TestViewResize {
 
 struct Default {};
 struct WithoutInitializing {};
+struct NoDefaultConstructor {
+  NoDefaultConstructor(int) {}
+};
```

Comment:
Where is that guy used?  Did you forget to commit something?

Reply:
Yeah, this is an oversight. I tried testing resize too, but could not trigger it.

Reply:
Let's add the test anyway.  More coverage is not going to hurt.